server/services/visitSessionsService.ts

```diff
@@ -205,14 +209,46 @@ export default class VisitSessionsService {
     const token = await this.systemToken(username)
     const visitSchedulerApiClient = this.visitSchedulerApiClientBuilder(token)
 
-    const visits = await visitSchedulerApiClient.getUpcomingVisits(offenderNo)
     logger.info(`Get upcoming visits for ${offenderNo}`)
+    const visits = await visitSchedulerApiClient.getUpcomingVisits(offenderNo)
 
     return visits.map(visit => {
       return this.buildVisitInformation(visit)
     })
   }
 
+  async getFullVisitDetails({
+    username,
+    reference,
+  }: {
+    username: string
+    reference: string
+  }): Promise<{ visit: Visit; visitors: Contact[] }> {
+    const token = await this.systemToken(username)
+    const visitSchedulerApiClient = this.visitSchedulerApiClientBuilder(token)
+    const prisonerContactRegistryApiClient = this.prisonerContactRegistryApiClientBuilder(token)
+
+    logger.info(`Get visit ${reference}`)
```

Comment:
Loggers might be better in the client? That way you'd get the log on every call? Pretty sure the REST client logs every request anyway but better in client rather than having to do this everywhere it's called?

Reply:
Some stray debugging - will remove!